requirements.txt

```diff
@@ -14,7 +14,7 @@ mistune>=0.8.4
 nbformat>=5.0
 notebook>=6.4.10
 numpy>=1.19.5; python_version == "3.8" or python_version == "3.9"
-numpy>=1.23.0; python_version >= "3.10"
+numpy>=1.21.2; python_version >= "3.10"
```

Comment:
You will need to update `ci/constraints-test/py310-min-install.txt` so we run tests against this minimum version.
Also, have you looked at usage stats for say python 3.9 users to see if they are often using this lower version of pandas?

Reply:
Talked to @jcampbell. AWS managed airflow pins numpy to a lower version than we currently support so we want to drop this version to support that.

Reply:
Updated the PR description to better capture the issue, and the version to focus narrowly on compatibility.